test/support/provider_test/comprehensive.ex

```diff
@@ -488,7 +503,7 @@ defmodule ReqLLM.ProviderTest.Comprehensive do
               {:ok, response} =
                 ReqLLM.generate_object(
                   @model_spec,
-                  "Generate a software engineer profile",
+                  "Generate a fictional software engineer profile with a made-up name and age",
```

Comment:
@shelvick this change is a deal breaker, as it will force redoing all of our fixture tests